model.go

```diff
@@ -90,6 +90,13 @@ func (f *FlagModel) FormatPlaceHolder() string {
 	return strings.ToUpper(f.Name)
 }
 
+func (f *FlagModel) HelpWithEnvar() string {
+	if f.Envar == "" {
+		return f.Help
+	}
+	return fmt.Sprintf("%s (env var: %s)", f.Help, f.Envar)
```

Comment:
Instead of `(env var: %s)` just make this `($%s)`, it's shorter and still clear.

Reply:
Done and rebased.